SPECS/kernel/config_aarch64

```diff
@@ -2981,7 +2981,8 @@ CONFIG_MII=y
 CONFIG_NET_CORE=y
 CONFIG_BONDING=m
 CONFIG_DUMMY=m
-# CONFIG_WIREGUARD is not set
+CONFIG_WIREGUARD=m
```

Comment:
Can you add a link to the kconfig for this config in the description?

Reply:
I added a link to kconfig